.github/workflows/build.yml

```diff
@@ -21,32 +21,30 @@ jobs:
       matrix:
         os:
           - ubuntu-latest
-          - windows-latest
 
         php:
-          - "7.4"
-          - "8.0"
+          - 7.4
+          - 8.0
+
+        pgsql:
+          - 13
+
+    services:
+      postgres:
+        image: postgres:${{ matrix.pgsql }}
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: testdb
+        ports:
+          - 5432:5432
+        options: --name=postgres --health-cmd="pg_isready" --health-interval=10s --health-timeout=5s --health-retries=3
```

Comment:
Because you need to test it with a specific driver, you should either test them with all or just sqlite like this, when a test passes one driver it passes all, and I can guarantee that.

Reply:
`sqlite` does not support everything that needs to be tested.